static/app/types/system.tsx

```diff
@@ -115,6 +115,7 @@ declare global {
 export interface Config {
   apmSampling: number;
   csrfCookieName: string;
+  customerDomain: string | undefined;
```

Comment:
from the backend pr it looks like it could be null
```suggestion
  customerDomain: string | null;
```